README.md

```diff
@@ -1,38 +1,343 @@
+|-------------------|-------------------------------------------------------------------------------------------------------------------------|
+| LOG_CHANNEL       | The channel for the `Audit Log` module to post to                                                                       |
+| ADMIN_LOG_CHANNEL | the channel for the  `Admin Loggers` modules to post to                                                                 |
+| JAIL_CHANNEL      | the channel you use to `jail` members                                                                                   |
+| BIRTHDAY_CHANNEL  | Used in the `birthday` command to wish members happy birthday, if this setting is not set, the command will be disabled |
+
 ## <ins>Modules</ins>
 
-There are 12 main modules in this bot, each one has its own set of permissions as defined by the`CommandSecurityModels`
-database table.
-
-| Module | Description |
-| --- | ----------- |
-| Ages | commands to get ages of accounts and servers |
-| ResourceBanner | Commands deal with banning attachments, embeds and emojis from messages |
-| Mute | Commands to mute people from servers |
-| Username | Commands to set usernames for people|
-| Memes | Commands generate memes. All commands that are marked as type 'attachment' can take both an uploaded image (has to be jpg) OR a mention |
-| Bookmarks | Commands to manage bookmarks |
-| Help | Commands to display help and info |
-| messageschedule | Commands to schedule posts to channels|
-| Miscellaneous | Miscellaneous commands |
-| closeableModules | Commands to enable or disable modules and commands |
-| Music | Commands to play music from Youtube |
-| RoleJoiner | Commands to allow users to join vanity roles|
-| Settings | Commands to change internal settings of this bot |
-
-you may use the`~help`command for more info about each command and how to call each command complete with mandatory
-arguments and examples.
+All modules in this application are defined by implementing the `ICloseableModule` interface. this applies to events
+only, not commands. Commands are defined in the own entity set contained in categories.
+
+### <ins>events</ins>
+
+#### Audit Logger
+
+The Audit logger is a non-admin styled logger designed to log quick events like when a member leaves, joins, kicked and
+banned (and by whom) and when an Auto mod event is triggered.
+
+In order for this module to work, it must be both enabled AND have the `LOG_CHANNEL` global setting populated with the
+ID of a channel
+
+This is a quick non-admin style logger that can log:
+
+- Member join
+- Member kick
+- Member leave
+- Member muted
+- Member un-muted
+- Member ban
+
+***
+
+#### Admin Loggers
+
+Admin Loggers are an embed style logger that will post realtime, detailed events to a chosen channel, these events are
+presented in a rich-embed. All triggered events contain an executor, in that will tell you who did what.
```

Comment:
Not entirely sure what an executor is in this context, However Im gonna presume it's the function that handles the logging aspect of the event, If this is the case maybe try: 

"All triggered events contain an executor, the executor will tell you who did what"

Reply:
the `executor` is the person who triggered the event. if i delete your message on discord, than I am the executor, it is not my terminology, it comes from discord API

Reply:
> Not entirely sure what an executor is in this context, However Im gonna presume it's the function that handles the logging aspect of the event, If this is the case maybe try:
> 
> "All triggered events contain an executor, the executor will tell you who did what"

how about

```
Admin Loggers are an embed style logger that will post realtime, detailed events to a chosen channel, these events are
presented in a rich-embed. All triggered events contain an executor, the executor is the member who triggered the event;
for example, if i deleted your message in Discord, then i would be the executor
```
?

Reply:
Sounds awesome, Not sure if you need the example, up to you if you wanna keep the example, depends if you have already provided an example else where

Reply:
I provide example images soon, so i will remove it, but for now, i keep

Reply:
Okii